runestone/assess/assess.py

```diff
@@ -32,11 +32,12 @@ def setup(app):
     app.add_directive("qnum", QuestionNumber)
     app.add_directive("timed", TimedDirective)
 
-    app.add_config_value("mchoice_div_class", "runestone alert alert-warning", "html")
+    app.add_config_value('mchoice_div_class', 'runestone alert alert-warning', 'html')
```

Comment:
There should be left unchanged -- it looks like code that black processed in the original vs. no-formatted code in the diff.